src/protocols/bitcoin/libs/BitcoinAdapter.ts

```diff
@@ -40,6 +40,10 @@ export class BitcoinAdapter extends BaseProtocolAdapter {
     },
   ];
 
+  override getAmountDecimalPlaces(): number {
+    return BTC_COIN_PRECISION;
```

Comment:
I think we need to maintain one naming convention `precision` or `decimalPlaces`.